src/NzbDrone.Core/MediaFiles/AudioTagService.cs

```diff
@@ -76,6 +80,15 @@ public AudioTag GetTrackMetadata(TrackFile trackfile)
             var albumartist = album.Artist.Value;
             var artist = track.ArtistMetadata.Value;
 
+            var cover = album.Images.FirstOrDefault(x => x.CoverType == MediaCoverTypes.Cover);
+            string imageFile = null;
+            if (cover != null)
+            {
+                imageFile = _mediaCoverService.GetCoverPath(album.Id, MediaCoverEntity.Album, cover.CoverType, 250);
```

Comment:
Should we use full size image for embed (or does file size become an issue)? Looking around many people prefer 500x500px up to 1000x1000px. Personally, I used to always shoot for 500px square back in the day, but I hate to add another standard resize to album covers, as it just slows down everything during refresh.